packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js

```diff
@@ -31,6 +31,12 @@ class SqlInjectionAnalyzer extends InjectionAnalyzer {
 
     this.addSub('datadog:mysql:pool:query:start', ({ sql }) => this.getStoreAndAnalyze(sql, 'MYSQL'))
     this.addSub('datadog:mysql:pool:query:finish', () => this.returnToParentStore())
+
+    this.addSub('datadog:knex:raw:start', ({ sql, dialect: knexDialect }) => {
+      const dialect = this.normalizeKnexDialect(knexDialect)
```

Comment:
I think that we should do it in the instrumentation file `knex.js` instead of in the subscription.

Reply:
The normalization is done to match the correct values in IAST evidence redactor.

I think we should leave as it is, to keep `knex` instrumentation agnostic and let subscribers manage the value in its own way if necessary.